packages/block-library/src/list/transforms.js

```diff
@@ -65,19 +65,6 @@ const transforms = {
 				} );
 			},
 		},
-		{
-			type: 'block',
-			blocks: [ 'core/quote', 'core/pullquote' ],
```

Comment:
No need to handled this anymore. The quote block now has a wildcard transform which includes wrapping a list into a quote.